mod-pubsub-client/src/main/java/org/folio/rest/util/OkapiConnectionParams.java

```diff
@@ -13,12 +13,9 @@
   private String tenantId;
   private String token;
   private Map<String, String> headers;
-  private Vertx vertx;
+  private final Vertx vertx;
   private int timeout = 2000;
 
-  public OkapiConnectionParams() {
-  }
-
   public OkapiConnectionParams(Vertx vertx) {
     this.vertx = vertx;
   }
```

Comment:
use Lombok

Reply:
Why?

Reply:
@adamdickmeiss this was suggested by the team during code review.  It is just to avoid boilerplate code, nothing else. Should I remove it?